src/components/schedule-page/validation-drawer/validation-drawer.component.tsx

```diff
@@ -1,28 +1,20 @@
-import { Drawer, List, ListItem, ListItemIcon, ListItemText } from "@material-ui/core";
-import ErrorOutlineIcon from "@material-ui/icons/ErrorOutline";
+import Drawer from "../../common-components/drawer/drawer.component";
 import React, { useEffect, useState } from "react";
 import { useSelector } from "react-redux";
 import { ApplicationStateModel } from "../../../state/models/application-state.model";
-import {
-  ScheduleErrorLevel,
-  ScheduleErrorMessageModel,
-} from "../../../common-models/schedule-error-message.model";
+import { ScheduleErrorMessageModel } from "../../../common-models/schedule-error-message.model";
 import { SpanErrors } from "./span-errors.component";
+import ErrorList from "./error-list.component";
 
 export default function ValidationDrawerComponent(): JSX.Element {
   const [mappedErrors, setMappedErrors] = useState<ScheduleErrorMessageModel[]>();
   const [open, setOpen] = useState<boolean>(false);
   const { scheduleErrors } = useSelector((state: ApplicationStateModel) => state);
 
   useEffect(() => {
-    const noErrorsFound = [
-      { kind: "", message: "Nie znaleziono błędów", level: ScheduleErrorLevel.INFO },
-    ];
     if (scheduleErrors?.length) {
       setMappedErrors(scheduleErrors);
       setOpen(true);
-    } else {
-      setMappedErrors(noErrorsFound);
     }
   }, [scheduleErrors]);
```

Comment:
Moim zdaniem powinniśmy zmienić tę logikę w taki sposób, żeby naciśnięcie przycisku otwierało drawer i jeżeli nie otrzymaliśmy jeszcze odpowiedzi z backendu wyświetlać wewnątrz niego spiner (kręcace się kółeczko).
Takie rozwiązanie wydaje mi się lepsze, ponieważ wtedy wiemy co się faktycznie dzieje. Teraz miałem problem kiedy nie włączyłem sobie serwera, a drawer się nie pojawiał. Dodatkowo jeżeli odpowiedź z serwera dojdzie z opóźnieniem to może to być trochę nieintuicyjne, że nagle otwiera nam się drawer. 
Oczywiście nie jest to częścią tego zadania, więc daję approve, ale myślę, że mogłabyś @Ehevi zajać się tym podczas kolejnej iteracji, kiedy będziesz wprowadzać odpowiednie nagłówki błedów. 
Co o tym myślisz @Qwebeck? 

Zamieszam instrukcję jakbym ja to zrobił, oczywiście to tylko sugestia. Jeśli będziesz to robić to zrób to jak Ty uważasz. 
Najpierw trzeba by usunąć hooka `useEffect`, ponieważ to on odpwiada za rozuwanie drawera, kiedy zmienią się błędy (przekazanie do niego w drugim argumencie `[scheduleErrors]` powoduje wywołanie funkcji jeżeli błędy zostaną zaktualizowane). Wtedy rodzic musi trzymać stan drawera i przekazywać go do niego, a funkcję która ma ustawić stan należy przekazać do przycisku, który ma go otwierać. Można tutaj wzorować się na tym jak to zostało zrobione w `WorkersTab` dla `WorkerDrawerComponent`

Reply:
@pectom , @Ehevi 

> Oczywiście nie jest to częścią tego zadania, więc daję approve, ale myślę, że mogłabyś @Ehevi zajać się tym podczas kolejnej iteracji, kiedy będziesz wprowadzać odpowiednie nagłówki błedów.
Co o tym myślisz @Qwebeck?

Tak. Takie zachowanie było dodane w innym PR, nie było dopracowane, bo chieliśmy to zakmknąć przed końcem sprintu.

> Najpierw trzeba by usunąć hooka useEffect, ponieważ to on odpwiada za rozuwanie drawera, kiedy zmienią się błędy (przekazanie do niego w drugim argumencie [scheduleErrors] powoduje wywołanie funkcji jeżeli błędy zostaną zaktualizowane).

Ja bym zostawił, tylko zmieniłbym logikę, bo musimy w jakiś sposób podmieniać błędy i kółko?

> Wtedy rodzic musi trzymać stan drawera i przekazywać go do niego, a funkcję która ma ustawić stan należy przekazać do przycisku, który ma go otwierać

Nie jestem do końca pewny czy o to ci chodziło, ale ja bym przywrócił przycisk pokaż błędy. Który by dzialał tak jak wcześnie, tylko być może miałby inny napis.